apollo-federation/src/query_plan/query_planner.rs

```diff
@@ -339,6 +346,7 @@ impl QueryPlanner {
         &self,
         document: &Valid<ExecutableDocument>,
         operation_name: Option<Name>,
+        options: Option<QueryPlanOptions>,
```

Comment:
```suggestion
        options: QueryPlanOptions,
```

Since `None` means "use defaults", can we just use `QueryPlanOptions::default()` at call-sites?

Reply:
👍 yeah while porting it over I tried to match JS version where this is optional.... but in JS it results in overloaded method where we can pass parameter or not. In RS we always have to pass it so doesn't make sense. I'll change it up.